cash-bail-and-race.html

```diff
@@ -36,12 +36,10 @@
               <div class="default text">Search counties</div>
               <div class="menu"></div>
             </div>
-            <div class="btn-text outliers-btn">Show Outliers</div>
           </div>
           <div class="switch-container">
-            <div class="btn-text switch left showing">% Cash Bail</div>
-            <div class="btn-text switch right">Bail Amount</div>
-            <div class="btn-text outliers-btn mobile-only">Show Outliers</div>
+            <div class="btn-text switch left showing">Cash bail rt.</div>
```

Comment:
This is inconsistent with `bail-trends`, which is "Cash Bail" (capitalized, no "rt."). We're okay with that?